src/Executables/Game/GameServer.cs

```diff
@@ -163,7 +170,7 @@ private async ValueTask Tick()
             if (_accumulatedElapsedTime < _targetElapsedTime)
             {
                 var sleepTime = (_targetElapsedTime - _accumulatedElapsedTime).TotalMilliseconds;
-                await Task.Delay((int) sleepTime).ConfigureAwait(false);
+                await Task.Delay((int)sleepTime).ConfigureAwait(false);
```

Comment:
I think we need an .editorconfig 👀

Reply:
Oops, wasn't paying attention

Reply:
You probably didn't do it on purpose. It just shows that this project has no .editorconfig yet